lib/drab/element.ex

```diff
@@ -288,6 +288,29 @@ defmodule Drab.Element do
     set_prop!(socket, selector, %{"style" => Map.new(properties)})
   end
 
+  @doc """
+  Broadcasting version of `set_style/3`.
+
+  It does exactly the same as `set_style/3`, but instead of pushing the message to the current
+  browser, it broadcasts it to all connected users.
+
+  Always returns `{:ok, :broadcasted}`.
+
+  See `Drab.Core.broadcast_js/2` for broadcasting options.  
+  """
+  @spec broadcast_style(Drab.Core.subject(), String.t(), map | Keyword.t()) :: Drab.Core.result()
+  def broadcast_style(subject, selector, properties) when is_list(properties) or is_map(properties) do
+    broadcast_js(subject, set_js(selector, %{"style" => Map.new(properties)}))
+  end
+
+  @doc """
+  Exception-throwing version of `broadcast_style/3`
+  """
+  @spec broadcast_style!(Drab.Core.subject(), String.t(), map | Keyword.t()) :: Drab.Core.return() | no_return
```

Comment:
There is no need to have bang versions of broadcasting functions, as they work in "fire and forget" regime.

Reply:
ok. I saw that the bang version for broadcast_js exist so I thinked it was better to add these versions

Reply:
True, it should not be there, I am going to depreciate this.